devel/gitlab-runner/Portfile

```diff
@@ -7,7 +7,7 @@ go.setup            gitlab.com/gitlab-org/gitlab-runner 14.5.1 v
 
 categories          devel
 platforms           darwin
-maintainers         {breun.nl:nils @breun} openmaintainer
+maintainers         openmaintainer
```

Comment:
this should become `nomaintainer` as `openmaintainer` is only used when the port is maintained

Reply:
Ah right, I've fixed that now.